deployments/rialto/docker-compose.git.yml

```diff
@@ -46,22 +46,22 @@ services:
         BRIDGE_HASH: ${RELAY_BRIDGE_HASH:-master}
         PROJECT: ethereum-poa-relay
 
-  bridge-node-alice: &bridge-node
+  rialto-bridge-node-alice: &rialto-bridge-node
```

Comment:
We'll need another deployment folder for Millau (not necessary in this PR), which will have it's own Docker Compose file. With different `docker-compose` files we don't need to have `rialto-*` in the names (they'll get added by docker based off the folder name, which in this case is `rialto`).

Reply:
The idea is to get rid of simple `bridge-node`, `bridge-runtime` everywhere. Now we don't have "node" and "runtime", we have either Rialto, or Millau "-node" and "-runtime".

About deployment - I'm not yet sure, but I'd prefer to bridge real Rialto-Sub chain - i.e. the chain that is syncing Rialto-PoA headers with Millau-Sub chain. There are two benefits imo:
1) we'll see how `HeaderChain`, `MessageLane`, `CurrencyExchange` transactions are competing for block in-the-wild;
2) we have (almost) complete application that'll work on top of `MessageLane` module - i.e. we may change our currency-exchange-transactions-relay to work with `MessageLane` module of Millau-Sub. So that it'll:
2.1) will read proof-of-exchange transactions from Rialto-PoA node;
2.2) will submit Rialto-Sub **calls** as **messages** to Millau-Sub `MessageLane` module;
2.3) message-lane relay would finally submit these messages to the Rialto-Sub `MessageLane` module.

Reply:
Yeah maybe we need to rethink our deployments a little. Maybe something like this?

```
deployments
├── rialto // Eth <-> Sub
├── millau // Sub <->  Sub
└── tbd    // Eth <-> Sub <-> Sub
```

Each one of these would have their own compose files, leveraging the same published images. This doesn't have to be done now (and can't since Sub<->Sub is still not a thing), but something to keep in mind.

In this setup though I don't think any of the nodes aside from the `tbd` ones would need to have prefixes in the names.

Reply:
I think what we need is `deployments/rialto` and `deployments/millau` with two `docker-compose` files, which will be using the same docker network => they both should have prefixes. But it is undecided => I do not want to talk about it now.

What's wrong with having `rialto-` prefix? Why we're using `bridge-node-` prefix then? I don't want to see `bridge-node` and `bridge-runtime` anymore, simply because we do not have `bridge-node` and `bridge-runtime` crates anymore.

Reply:
Right now if I do `docker ps -a` I get back container names that look like this:

```
rialto_bridge-node-alice_1
```

If we add the `rialto` prefix our logs will end up looking like `rialto_rialto-bridge-node-alice_1` which is kinda awkward. If we leave them as they are in the compose file now they'll be prefixed correctly based off their folder.

Reply:
Ok, this makes sense. I'm removing `rialto-` prefix from container name, so logs should not have this duplicated prefix. I've added network aliases for default network instead (later we'll probably switch to some persistent network => we only have to change that "default:" in `docker-compose.yml`). I've also added optional `hostname:` association (thought it'll solve problem initially).

Reply:
I haven't used `networks` in Docker yet, so I don't know how that'll look in practice, but on the surface it looks alright